.github/workflows/test_code.yml

```diff
@@ -11,16 +11,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.11"]
-        # We only run latest Python on macOS and Windows to save resources.
-        include:
-          - os: ubuntu-latest
-            python-version: "3.8"
-          - os: ubuntu-latest
-            python-version: "3.9"
-          - os: ubuntu-latest
-            python-version: "3.10"
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Any specific reason why you dropped macOS from the matrix?

Reply:
We're not running those on Folium as well. I'd like to make Folium and Branca more consistent, so it's easier to update both. And I figured with a Linux and Windows test, we got the two main flavors covered. Would you agree or do you think we should keep Macos and add it to the Folium tests as well?

Reply:
Thanks Martin, appreciate you looking out!